client/ayon_harmony/api/js/AyonHarmonyAPI.js

```diff
@@ -1,22 +1,22 @@
 // ***************************************************************************
-// *                        Avalon Harmony Host                                *
+// *                        AYON Harmony Host                                *
 // ***************************************************************************
 
 
 /**
  * @namespace
- * @classdesc AvalonHarmony encapsulate all Avalon related functions.
+ * @classdesc AyonHarmonyAPI encapsulate all AYON related functions.
  */
-var AvalonHarmony = {};
+var AyonHarmonyAPI = {};
 
 
 /**
  * Get scene metadata from Harmony.
  * @function
  * @return {object} Scene metadata.
  */
-AvalonHarmony.getSceneData = function() {
-    var metadata = scene.metadata('avalon');
+AyonHarmonyAPI.getSceneData = function() {
+    var metadata = scene.metadata('ayon');
```

Comment:
This will not work on legacy files.

Reply:
How about this? https://github.com/ynput/ayon-harmony/pull/14/commits/55228cc3a9fdfece00b7163fd95fb2af4b7d2466